MainTel/MainTel-Troubleshooting/MTCS-Troubleshooting/TMA-troubleshooting/TMA-Troubleshooting-OSS-Fails-to-Turn-On-index.rst

```diff
@@ -0,0 +1,181 @@
+    :name: alarm menu
+
+
+
+
+
+The large grey box on the cabinet's side (purple arrow) shows a blinking red light and an interlock in the screen.
+
+
+
+.. figure:: ./_static/grey_box_cabinets.jpeg
+    :name: grey box cabinets
+
+    .. figure:: ./_static/interlock_screens.jpeg
+        :name: interlock screens
+
+3. There are two reset buttons inside the chiller cabinet that needs to be pressed. From the side of the cabinet, use the screwdriver to remove two screws that are located on top. This will allow you to remove the cover panel.
+ Each cabinet has two panels. In chiller #2 the reset buttons are within the furthermost panel (Image below), while in chiller#1 they are found under the panel closer to the front face of the chiller.
+
+.. figure:: ./_static/furthermost_panel.jpeg
+    :name: remove panel
+
+    .. figure:: ./_static/Hand.jpeg
+        :name: 
+
+
+4. Press each of the red reset buttons inside the cabinet for 3 seconds (TBC: order does not matter):
+
+.. figure:: ./_static/reset_button_1.jpeg
+ :name: buttons
+
+
+
+.. figure:: ./_static/reset_button_2.jpeg
+ :name: reset
+
+
+
+
+   .. _Title-of-Troubleshooting-Procedure-Final-Step:
+
+
+
+5. Press the reset button on the large grey box on the side of the cabinet. The red alarm light should stop blinking.
+
+6. Acknowledge the alarms by pressing the tick mark on the bottom of the screen. After perhaps a minute or two, the front screen should show **"STATUS:OK"**
+
+.. figure:: ./_static/acknowledge_alarm.jpeg
+    :name:
+
+7. Repeat **steps 2-6** with the other chiller, if it shows an alarm.    
+
+8. Back to the control room, reset the alarm in the TMA EUI **"Safety System"** screen. Click on the red **"OSS malfunction"** and press **"RESET SELECTED"**. The red colour should dissapear.
+
+.. figure:: ./_static/safety_system.png
+    :name:
+
+
+
+
+.. _Title-of-Troubleshooting-Procedure-Condition-A-for-Step-4:
+
+
+
+Post-Condition
+==============
+
+.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Please provide screenshots of the software status or relevant display windows to confirm.
+.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
+
+- The system should be now ready to Turn On the OSS again.
+
+
+
+.. _Title-of-Troubleshooting-Procedure-Contingency:
+
+Contingency
+===========
+
+If the procedure was not successful, report the issue in channel **summit-simonyi** and/or activate the :ref:`Out of hours support <Safety-out-of-hours-support>`.
```

Comment:
The channel doesn´t need to be in bold, but it does need an # before its name. #summit-simonyi